### DISCUSSION TITLE
Support for just announced Pan V3?

### DISCUSSION THREAD

**Discussion by rvs007** [gtxaspec/wz_mini_hacks#457] [General]
The Wyze Cam Pan V3 just got announced... when will we know if the wz_mini hack will support this model?

https://ca.wyze.com/products/wyze-cam-pan-v3

**Comment by endertable** [comment]
Just ordered one yesterday  so we can find out soons i get my grimy hands on it.  :)

**Reply by KyleStilkey** [reply]
I ordered one as well, my guess is this is already patched out, but I will see what it ends up coming with and attempting to see if anything works, but really going to say no.

**Reply by endertable** [reply]
Agreed.  My guess is they will be using the same firmware as the pro V3, which has not been Infiltrated as of yet

**Comment by gtxaspec** [comment]
Just got one today, let's break it open and see.

**Reply by endertable** [reply]
Very good!  Can’t wait to hear your initial thoughts!

**Comment by jmbulan** [comment]
any update? i have one in my amazon cart just wating to see if itll work lol

**Comment by mach7** [comment]
@gtxaspec @endertable how goes the new pan cam?

**Reply by endertable** [reply]
Its a fine lil cam.  Faster and covers mores area than its predecessors.  As far as hackability goes, i havent heard of anyone cracking it, yet.

**Reply by mach7** [reply]
Have you personally tried hacking yours? Sounds like you have one. I'd like to get one, but I'd like to have wz_mini on it if possible.

**Reply by endertable** [reply]
Developer of wz_mini confirmed that currently the mod does not work with pan cam v3, nor the new OG series of cams.
You can also include V3 Pro not compatible with this hack either. At least not yet..

**Reply by mach7** [reply]
From what I've read it looks like they're switching to some different internals and firmware. So the days of utilizing these hacks may be numbered. Which stocks because I have dove head first and started to replace everything I had camera wise with the v3. Maybe it's time to start using the funds to research some different cameras that will fit my needs.

**Reply by endertable** [reply]
The latest isnt always the greatest.  The V3 and the pan cam V2 do work with this mod and they are very formidable devices that can provide years of good service.   You wont find many other offerings that give as much as this combo anywhere near this price range.  I have some Sony cams that have full PTZ with 25x zoom and can be viewed by just about anything, including browsers, but they run over $800 a piece, used.
TBH, the only kind of cam that Wyze could release that would have me replace my V3s would be one with a zoomable lense, not the OG combo thing they have now.
This is the year of cameras for them, so lets see…

**Comment by endertable** [comment]
Developer of wz_mini confirmed that currently the mod does not work with pan cam v3, nor the new OG series of cams.  
You can also include V3 Pro not compatible with this hack either.  At least not yet..
